setup.py

```diff
@@ -20,7 +20,7 @@
 
 setup_kwargs = {
     'name': 'exasol-script-languages-container-ci',
-    'version': '0.6.0',
+    'version': '1.0.0',
```

Comment:
Maybe mark this as generated in `.gitattributes`

Reply:
done